### PR TITLE
helpers consolidated to utilities

### PR DESCRIPTION
**Pull request by sarupanda**

### CODE REVIEW

File: hello_bot/hello_bot.py [hunk 1]
```diff
@@ -10,27 +10,21 @@
...
-    # has been sent and therefore needs processing.
+    # Only execute this section of code when a POST request is sent, as a POST indicates when a
+    # message has been sent and therefore needs processing.
     if request.method == 'POST':
         json_data = request.json
-        print("\n")
-        print("WEBHOOK POST RECEIVED:")
-        print(json_data)
-        print("\n")
+        print(f"\nWEBHOOK POST RECEIVED: {json_data}\n")
```
Comment (paralta) [inline hello_bot/hello_bot.py:27]: perfect!

File: hello_bot/hello_bot.py [hunk 2]
```diff
@@ -58,10 +52,7 @@ def teamswebhook():
 
 
 if __name__ == '__main__':
-
-    # Read the configuration that contains the bot access token
-    config = read_yaml_data(os.getcwd() + '\config\config.yaml')['hello_bot']
-    teams_api = WebexTeamsAPI(access_token=config['teams_access_token'])
+    teams_api = WebexTeamsAPI(access_token='')
```
Comment (paralta) [inline hello_bot/hello_bot.py:55]: We can discuss later if the access token should be a global variable at the beginning of this file or we should add instructions to add it here

**Review by paralta — APPROVED** [review]
LGTM

File: hello_bot/hello_bot.py (hunk 2 above)
Comment (sarupanda) [reply hello_bot/hello_bot.py:55]: It depends on what happens if you make multiple WebexTeamsAPI objects with the same token. Does it still work?

**Pull request merged**